main.go

```diff
@@ -87,9 +105,19 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Cache: cache.Options{
+			ByObject: map[client.Object]cache.ByObject{
+				&tektonv1.PipelineRun{}: cache.ByObject{
+					Label: labels.SelectorFromSet(labels.Set{metadata.ServiceNameLabel: metadata.ServiceName}),
```

Comment:
Is it possible to put an OR in here and then just reuse the managed, tenant, and final pipeline labels without having to add a new one?

Reply:
It allows a list of Labels. So your idea would be we filter the value by `metadata.TenantPipelineType`,  `metadata.ManagedPipelineType` or `metadata.FinalPipelineType` ? Is that correct?

Reply:
I don´t understand much about the collectors, but sounds like a single label would make it easier. I don´t see (at least so far) a real need for different labels for the cache. Unless we decide in the future we want to cache only  managed, for instance. 

So far it seems enough a single one, even though we are having to add it.  For the moment I would just leave it as it looks now and change it if we understand it is required.